tests/install/CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.21)
-project(asio-cmake-install)
-find_package(asio)
+project(asio-cmake-install LANGUAGES CXX)
+find_package(asio REQUIRED)
```

Comment:
Good idea it gives better error message than if it fail to find target `asio::asio`